tests/test_nanopub.py

```diff
@@ -9,96 +9,85 @@
 from nanopub import NanopubClient, namespaces
 from nanopub.nanopub import Nanopub
 
-DEFAULT_FORMAT = '.trig'
-BAD_GATEWAY = 502
-NANOPUB_SERVER = 'http://purl.org/np/'
-SERVER_UNAVAILABLE = 'Nanopub server is unavailable'
+skip_if_nanopub_server_unavailable = (
+    pytest.mark.skipif(requests.get('http://grlc.nanopubs.lod.labs.vu.nl/').status_code != 200,
```

Comment:
This should be replaced with the test server later.

Reply:
For testing publishing, yes, although for simple fetching of existing nanopubs at known URIs it might be worth still using the real servers for that, just to indicate to us if something there has changed.

Reply:
Let's discuss this on the test endpoint PR.